src/screens/OParl/OParlPersonsScreen.tsx

```diff
@@ -69,6 +72,7 @@ const useListData = (
     if (organization) {
       setListData(
         organization.membership
+          ?.filter((mem) => !mem.endDate)
```

Comment:
`mem` could be `membership` as well here like in the next line, because it is the same, right?

Reply:
renamed to mem